source/API/CMakeLists.txt

```diff
@@ -170,7 +170,16 @@ if(LLDB_BUILD_FRAMEWORK)
 endif()
```

Comment:
Related: framework builds have their own way of dealing with headers. This should be an `else()`? (I can check this soon.)

Reply:
Ugh, really?  There is a third way to deal with the headers? :/  Thanks for looking into that.